app/models/response/ResponseArrival.scala

```diff
@@ -18,15 +18,16 @@ package models.response
 
 import java.time.LocalDateTime
 
-import models.domain.{Arrival}
+import models.domain.Arrival
 import play.api.libs.json.Json
+import utils.Utils
 
 object ResponseArrival {
 
   implicit val format = Json.format[ResponseArrival]
 
   def apply(a: Arrival): ResponseArrival = {
-    ResponseArrival(a.location, a.created, a.updated, a.movementReferenceNumber, a.status, a.messagesLocation)
+    ResponseArrival(s"/movements/arrivals/${Utils.urlEncode(a.arrivalId.toString)}", a.created, a.updated, a.movementReferenceNumber, a.status, s"/movements/arrivals/${Utils.urlEncode(a.arrivalId.toString)}/messages")
```

Comment:
As it's a location, shouldn't the string start with "/customs/transits"? I think the get methods on the controllers  are missing this too.

Reply:
The /customs/transits prefix is only when we are in managed envs.  It's added by the API Platform so not needed for local testing.

Reply:
> The /customs/transits prefix is only when we are in managed envs. It's added by the API Platform so not needed for local testing.

I think it's still required in the response here. That way external users can use the path directly in their code without having to add the /customs/transits prefix.

Reply:
I won't lie, I'm against including it.
1) Because if we use the reverseRouting we don't get it.
2) Our root location is /customs/transits/ in the environments, and / locally. We don't include www.governmenturlstuff.com/ so why include the rest of the prelude?